Documentation/Books/Manual/Administration/Replication/README.md

```diff
@@ -5,21 +5,49 @@ Replication allows you to *replicate* data onto another machine. It
 forms the base of all disaster recovery and failover features ArangoDB
 offers. 
 
-ArangoDB offers asynchronous and synchronous replication which both
-have their pros and cons. Both modes may and should be combined in a
-real world scenario and be applied in the usecase where they excel
-most. 
+ArangoDB offers **asynchronous** and **synchronous** replication,
+depending on which type of arangodb deployment you are using.
 
-We will describe pros and cons of each of them in the following
-sections. 
+Since ArangoDB 3.2 the *synchronous replication* replication is the *only* replication
+type used in a cluster whereas the asynchronous replication is only available between
+single-server nodes. Future versions of ArangoDB may reintroduce asynchronous
+replication for the cluster.
+
+### Asynchronous replication
+
+In ArangoDB any write, delete or update operation will be logged to the write-ahead
+log (WAL), before beeing acknowledged to the client. The write ahead ensures
+that in the occurance of an unscheduled server shutdown, we are still
+able to restore already commited transactions.
+
+The asynchronous replication is used as the foundation of most replication operations in
+arangodb. Effecitvely you as a potential end-user will **only** ever encouter it directly
+**on single-instance**nodes.
+
+The WAL is also useful for other purposes than data-integrety, it also allows
+other replication clients to follow along with all changes made on a specific *leading* server.
+These async-replication clients are called *followers*, which will periodically poll the *leader*
+for changes to it's data. The clients will apply all modification operations in the same order
+on the local node. Assuming the initial dataset was identical to the dataset on the leader node,
+the follower nodes should eventually have the exact same state as the leader.
+
+The asynchronous replication allows you to achieve **eventual consistency** between one leader
+ArangoDB node and multiple follower nodes. For more information see the [relevant sections](Asynchronous/README.md)
 
 ### Synchronous replication
 
-Synchronous replication only works within a cluster and is typically
-used for mission critical data which must be accessible at all
-times. Synchronous replication generally stores a copy of a shard's
-data on another db server and keeps it in sync. Essentially, when storing
-data after enabling synchronous replication the cluster will wait for
+Synchronous replication is only available in the context of an ArangodB **cluster setup**.
+In the case of synchronous replication we are always talking about replicating **shards** of a collection
+(a shard is a fragment of a collection, stored on one DBServer node). Synchronous replication guarantees that
+replicated copies of a *leader* shard are stored on another *follower* db server and kept completely **in sync**.
+
+By "in-sync" we mean that we guarantees *strong consistency* between replica shards: Your data is
```

Comment:
"in-sync" (here) vs. "in sync" (in line above).
We should write it consistently.